src/server/game/Entities/Player/Player.h

```diff
@@ -2642,6 +2642,17 @@ class Player : public Unit, public GridObject<Player>
 
     static std::unordered_map<int, bgZoneRef> bgZoneIdToFillWorldStates; // zoneId -> FillInitialWorldStates
 
+    // Cinematic camera data and remote sight functions
+    uint32 GetActiveCinematicCamera() const { return m_activeCinematicCameraId; }
+    void SetActiveCinematicCamera(uint32 cinematicCameraId = 0) { m_activeCinematicCameraId = cinematicCameraId; }
+    bool IsOnCinematic() const { return (m_cinematicCamera != nullptr); }
```

Comment:
please add `[[nodiscard]]` to all getter methods that have no side effects

Reply:
@FrancescoBorzi  added !